lib/puppet/type/acl.rb

```diff
@@ -258,22 +258,24 @@ def should_to_s(shouldvalue)
       super(shouldvalue)
     end
 
-    def self.format_value_for_display(value)
-      if value.is_a? Array
-        formatted_values = value.collect {|value| format_value_for_display(value)}.join(', ')
-        "[#{formatted_values}\n]"
-      elsif value.is_a? Puppet::Type::Acl::Ace
-        "#{value.inspect}"
-      elsif value.is_a? Hash
-        hash = value.keys.sort {|a,b| a.to_s <=> b.to_s}.collect do |k|
-          "#{k} => #{format_value_for_display(value[k])}"
-        end.join(', ')
-
-        "\n { #{hash} }"
-      else
-        "'#{value}'"
-      end
-    end
+    # def self.format_value_for_display(value)
```

Comment:
Commented this just to see if it's actually called anywhere... its unused in specs on OSX, but maybe it is called on Windows somehow?